omniport/core/guest_auth/views/login.py

```diff
@@ -0,0 +1,63 @@
+import swapper
+from rest_framework import status, generics, response
+from django.contrib.auth import login
+from django.conf import settings
+
+from omniport.utils import switcher
+from base_auth.models import User
+from base_auth.managers.get_user import get_user
+from session_auth.models import SessionMap
+
+from guest_auth.utils.create_guest import create_guest
+
+Person = swapper.load_model('kernel', 'Person')
+
+AvatarSerializer = switcher.load_serializer('kernel', 'Person', 'Avatar')
+
+GUEST_USERNAME = settings.GUEST_USERNAME
```

Comment:
No point in reassigning after import, just use `settings.GUEST_USERNAME` wherever required.